app/controllers/sign_offs_controller.rb

```diff
@@ -3,7 +3,7 @@ class SignOffsController < ApplicationController
 
   def index
     @sign_offs =  if params[:filter_by].blank?
-                    SignOff.all.order(sort_column + ' ' + sort_direction).page(params[:page]).per(Settings.pagination.default)
+                    SignOff.all.order(sort_column + ' ' + sort_direction).page(params[:page]).per(10)
```

Comment:
`per(10)` If this is fixed then I think you can set this from model file. Is it possible?